### ISSUE TITLE
Support extra attributes on gtk4_widget macro

### ISSUE THREAD

**Issue by jf2048** [jf2048/gobject#26]
Things like `layout_manager_type`, `css_name`, `accessibility_role`. These should be usable without overriding `class_init`.
